htsget-http-actix/src/lib.rs

```diff
@@ -60,17 +35,17 @@ pub type HtsGetStorage = HtsGetFromStorage<LocalStorage>;
 #[cfg(feature = "async")]
 pub struct AsyncAppState<H: HtsGet> {
   pub htsget: Arc<H>,
-  pub config: Config,
+  pub config: HtsgetConfig,
```

Comment:
I'd leave it at `Config` instead of `HtsgetConfig`, is there a conflicting type/name? Even if there was, Rust idiomatic preference is variable shadowing, right?

Reply:
No conflict. I forgot why I changed it, but you're right, its better as just Config.